mavros/include/mavros/utils.h

```diff
@@ -65,6 +64,9 @@ std::string to_string(mavlink::common::MAV_AUTOPILOT e);
 std::string to_string(mavlink::common::MAV_TYPE e);
 std::string to_string(mavlink::common::MAV_STATE e);
 std::string to_string(timesync_mode e);
+std::string to_string(mavlink::common::ADSB_ALTITUDE_TYPE e);
+std::string to_string(mavlink::common::ADSB_EMITTER_TYPE e);
+std::string to_string(const uint16_t &bitmask);
```

Comment:
this is for debugging purposes. it's not for msg parsing. if you just tell it is not acceptable but don't suggest a solution, it's the same as nothing...